styleguide/_themes/derek/scss/patterns/blog-feed.scss

```diff
@@ -0,0 +1,71 @@
+@mixin blogFeed-textElements {
```

Comment:
Well its used in almost all of these elements. I used it so that in the future if we want to change how this text appears we can just change it here instead of going through all of those. I wouldve just used a class and extended it but sass-lint says thats a no-go. I included it within this file instead of a global file to avoid anyone using it for something other than this. so is there a better way? I felt like this was in the spirit of maintainable. Easy to update, module specific. ya know that stuff.